### PR TITLE
feat(prettier-config): expose configs for angular

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring
-   [ ] Code style update
-   [ ] Build or CI related changes
-   [ ] Documentation content changes

## What is the new behavior?

https://github.com/TinkoffCreditSystems/taiga-ui/issues/923

Support angular preset

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Comment by splincode** [comment]
@IKatsuba @waterplea please, review me))

**Comment by splincode** [comment]
@IKatsuba @SuperOleg39 @MarsiBarsi What do you think about PR? : )) any ideas

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
         "jest": "^26.0.1",
         "lerna": "^4.0.0",
         "lint-staged": "^10.0.1",
-        "prettier": "2.2.1",
+        "prettier": "^2.4.1",
```
Comment (SuperOleg39) [inline package.json:56]: prettier strongly recommended to use fixed versions of this package

File: packages/eslint-config/package.json [hunk 2]
```diff
@@ -34,7 +30,7 @@
         "eslint-plugin-prettier": "^3.1.4",
         "eslint-plugin-promise": "^4.2.1",
         "eslint-plugin-sort-class-members": "^1.7.0",
-        "prettier": "2.2.1"
+        "prettier": "^2.4.1"
```
Comment (SuperOleg39) [inline packages/eslint-config/package.json:33]: And here need to fix)

File: packages/prettier-config/README.md [hunk 3]
```diff
@@ -6,14 +6,29 @@ Common Prettier configuration for Tinkoff projects
...
 
-    ```bash
-    npm i --save-dev @tinkoff/prettier-config
-    ```
+```bash
+npm i --save-dev @tinkoff/prettier-config
+```
 
-1. Create `.prettierrc.js` at project root
+1. Add a "prettier" key in your package.json file.
```
Comment (SuperOleg39) [inline packages/prettier-config/README.md:13]: I think you need to write about prettierrc as alternative configuration variant

File: packages/prettier-config/prettier.config.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+module.exports = {
```
Comment (SuperOleg39) [inline packages/prettier-config/prettier.config.js:1]: Sorry, but you can't just make so huge changes, this config are used by tens of Tinkoff apps (react + tramvai), and it is breaking change, without motivation

**Review by SuperOleg39 — CHANGES_REQUESTED** [review]

File: packages/prettier-config/prettier.config.js (hunk 4 above)
Comment (splincode) [reply packages/prettier-config/prettier.config.js:1]: reverted

**Review by SuperOleg39 — APPROVED** [review]
LGTM

**Pull request merged**